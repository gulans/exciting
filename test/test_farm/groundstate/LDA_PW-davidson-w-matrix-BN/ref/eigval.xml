<?xml version="1.0" encoding="UTF-8"?>
<eigval>
  <kpt ik="1" vkl="0.000000000000e0 0.000000000000e0 0.000000000000e0">
    <state ist="1" eigenvalue="-6.338865587684e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-7.567818548063e-1" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-3.094112547458e-1" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="-1.574738327476e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="-1.574738300430e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="6.206936248169e-2" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="1.208286481470e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="1.519987802584e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="2" vkl="3.333333333333e-1 0.000000000000e0 0.000000000000e0">
    <state ist="1" eigenvalue="-6.338891393119e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-6.837284077550e-1" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-3.626321210685e-1" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="-2.425373520223e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="-2.088753415713e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="1.495025449201e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="1.874499446616e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="2.645768830931e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="3" vkl="6.666666666667e-1 0.000000000000e0 0.000000000000e0">
    <state ist="1" eigenvalue="-6.338891393119e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-6.837284077550e-1" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-3.626321210686e-1" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="-2.425373520223e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="-2.088753415713e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="1.495025449201e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="1.874499446616e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="2.645768830931e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="4" vkl="3.333333333333e-1 3.333333333333e-1 0.000000000000e0">
    <state ist="1" eigenvalue="-6.338883758310e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-6.170361482573e-1" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-4.076981096713e-1" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="-3.707393790069e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="-1.086889954342e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="5.883283393538e-2" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="3.381671525393e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="3.857640904480e-1" occupancy="0.000000000000e0"/>
  </kpt>
</eigval>
