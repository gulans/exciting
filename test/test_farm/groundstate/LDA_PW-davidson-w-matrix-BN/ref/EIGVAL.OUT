     4 : nkpt
     8 : nstsv

     1   0.000000000       0.000000000       0.000000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -6.338865588       2.000000000    
     2 -0.7567818548       2.000000000    
     3 -0.3094112547       2.000000000    
     4 -0.1574738327       2.000000000    
     5 -0.1574738300       2.000000000    
     6  0.6206936248E-01   0.000000000    
     7  0.1208286481       0.000000000    
     8  0.1519987803       0.000000000    


     2  0.3333333333       0.000000000       0.000000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -6.338891393       2.000000000    
     2 -0.6837284078       2.000000000    
     3 -0.3626321211       2.000000000    
     4 -0.2425373520       2.000000000    
     5 -0.2088753416       2.000000000    
     6  0.1495025449       0.000000000    
     7  0.1874499447       0.000000000    
     8  0.2645768831       0.000000000    


     3  0.6666666667       0.000000000       0.000000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -6.338891393       2.000000000    
     2 -0.6837284078       2.000000000    
     3 -0.3626321211       2.000000000    
     4 -0.2425373520       2.000000000    
     5 -0.2088753416       2.000000000    
     6  0.1495025449       0.000000000    
     7  0.1874499447       0.000000000    
     8  0.2645768831       0.000000000    


     4  0.3333333333      0.3333333333       0.000000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -6.338883758       2.000000000    
     2 -0.6170361483       2.000000000    
     3 -0.4076981097       2.000000000    
     4 -0.3707393790       2.000000000    
     5 -0.1086889954       2.000000000    
     6  0.5883283394E-01   0.000000000    
     7  0.3381671525       0.000000000    
     8  0.3857640904       0.000000000    

